exporters/logging/build.gradle

```diff
@@ -0,0 +1,10 @@
+description = 'OpenCensus Logging Exporter'
```

Comment:
probably all the exporters will be specific to trace or stats. Maybe name directories trace_logging and the package opencensus-trace-logging-exporter?